pytermgui/ansi_interface.py

```diff
@@ -12,17 +12,25 @@
 import re
 import sys
 import signal
+import asyncio
 
 from typing import Optional, Any, Union, Callable, Tuple
 from dataclasses import dataclass, fields
 from enum import Enum, auto as _auto
 from sys import stdout as _stdout
 from string import hexdigits
 from subprocess import run as _run, Popen as _Popen
-from os import name as _name, get_terminal_size, system
```

Comment:
Removed `get_terminal_size` function from **os** module since it would be better to import it from **shutil** which wraps it with helper functions with fallback capability